### PR TITLE
[kmac] Move to D2S

### PR DESCRIPTION
**Pull request by eunchan**
As all AIs have been addressed, KMAC moves to D2S officially.

Co-Authored-by: Michael Tempelmeier <michael.tempelmeier@gi-de.com>

### CODE REVIEW

File: hw/ip/kmac/data/kmac.prj.hjson
Comment (msfschaffner) [inline hw/ip/kmac/data/kmac.prj.hjson:18]: Is this a valid commit ID?

If you don't want to maintain the commit ID here, I believe it is fine to remove it.
In other IPs we only added the commit ID when we had to revert the version to a previous design stage, e.g.:
https://github.com/lowRISC/opentitan/blob/c3dc1fa938d1889e806ff531be305afcaeb52ad8/hw/ip/gpio/data/gpio.prj.hjson#L12-L27

**Review by msfschaffner — COMMENTED** [review]
LGTM, but it would be good to get explicit approval from DD and security reviewers (@rswarbrick, @vogelpi, @cdgori).

File: hw/ip/kmac/data/kmac.prj.hjson
Comment (eunchan) [reply hw/ip/kmac/data/kmac.prj.hjson:18]: Yep. That is the last commit that @m-temp pushed. I normally specified the commit id when the design stage has been changed.

Comment (msfschaffner) [reply hw/ip/kmac/data/kmac.prj.hjson:18]: Ah ok sounds good. I only looked at your last two commits in this PR since I thought you were referring to one of them - and they did not match :).

Comment (m-temp) [reply hw/ip/kmac/data/kmac.prj.hjson:18]: shouldn't we include your (two) fixes? So if we ever want to go back, they are covered?

**Comment by m-temp** [comment]
Just a nit, but there are still three ticks missing in #10823. Perhaps we simply forgot to check them off?

**Comment by eunchan** [comment]
Thanks for bring up them @m-temp I skimmed the issues and looks like everything can be closed #10760 will be close with this PR

**Comment by eunchan** [comment]
Made fix for #10760 into a separate PR.

**Review by rswarbrick — APPROVED** [review]

**Review by msfschaffner — APPROVED** [review]
I looked at the pending issues. This is good to go once https://github.com/lowRISC/opentitan/pull/11307 has landed (this is the fix for #10760).

File: hw/ip/kmac/data/kmac.prj.hjson
Comment (msfschaffner) [reply hw/ip/kmac/data/kmac.prj.hjson:18]: Yeah that is a good point. I would either do what @m-temp suggested, or remove the id entirely.

**Review by m-temp — APPROVED** [review]
LGTM

**Pull request merged**
